distributed/widgets/templates/cluster.html.j2

```diff
@@ -7,7 +7,7 @@
         <table style="width: 100%; text-align: left;">
             <tr>
                 <td style="text-align: left;">
-                    <strong>Dashboard:</strong> <a href="{{ dashboard_link }}">{{ dashboard_link }}</a>
+                    <strong>Dashboard:</strong> <a href="{{ dashboard_link }} target="_blank"">{{ dashboard_link }}</a>
```

Comment:
Thanks @ncclementi! 

> I'm still missing one case which is the LocalClusterlink, but I couldn't find where is the template with the href to the dashboard for the case of LocalCLuster

I think this is where the `LocalCluster` dashboard link comes from. I see there are two double-quotes at the end, but I think we only want one. Maybe that's what's causing the link to not open in a separate tab

Reply:
Specifically, I mean this

```suggestion
                    <strong>Dashboard:</strong> <a href="{{ dashboard_link }}" target="_blank">{{ dashboard_link }}</a>
```

Reply:
Great catch @jrbourbeau, thanks!